editor/styles/global.scss

```diff
@@ -188,5 +188,10 @@ to make sure it renders on top is a common fix */
   content: none !important;
 }
 
+/* Fixes the height and aspect ratio of toggle butons */
+.hide-toggle {
+  aspect-ratio: 1 !important;
```

Comment:
nice touch to handle this globally! it is always so tricky to get buttons with icons to look right!